timemachine/fe/free_energy.py

```diff
@@ -57,13 +57,21 @@ class MDParams:
     max_radius: float = 3.0  # nm
     freeze_reference: bool = True
 
+    # Number of frames to sample using MD during the initial bisection phase used to determine lambda spacing
+    hrex_n_frames_bisection: int = 100
+
+    # Number of frames to sample using MD per HREX iteration. Set to 0 to disable HREX.
+    hrex_n_frames_per_iter: int = 0
+
```

Comment:
nit: slight preference for defining a dataclass `HREXParams` to contain these fields, and including as `Optional[HREXParams]` here. Motivation would be to avoid some confusion (e.g. is `hrex_n_frames_bisection` ignored if `hrex_n_frames_per_iter == 0`?), and to avoid needing to use `0` as a special value (could just pass `HREXParams = None`)

Reply:
nit: change to documenting these fields as comments means they're no longer accessible in docstrings. Could consider documenting parameters of `MDParams` (or maybe just `HREXParams` if above suggestion is implemented?)